src/sage/combinat/crystals/spins.pyx

```diff
@@ -108,7 +108,7 @@ def CrystalOfSpinsPlus(ct):
     r"""
     Return the plus spin crystal of the given type D.
 
-    This is the crystal with highest weight `Lambda_n` (the
+    This is the crystal with highest weight `\lambda_n` (the
```

Comment:
The original paper (Crystal Graphs for Representations of the q-Analogue of Classical Lie Algebras
M. Kashiwara, Toshiki Nakashima) uses uppercase Lambda (Λ) for fundamental weight. (Edit the occurrence above too.)

Reply:
Thanks for the correction. I made the fix, so is it ok to restore to positive review?